pkg/storage/tsdb/block/index.go

```diff
@@ -559,10 +559,13 @@ OUTER:
 }
 
 func verifyChunks(cr *chunks.Reader, cm chunks.Meta) error {
```

Comment:
called from compactor which should deal with only non overlapping chunks so this is fine